### DISCUSSION TITLE
addSlide function can't add between slides

### DISCUSSION THREAD

**Discussion by Blaaszkovsky** [nolimits4web/swiper#7389] [Q&A]
### Check that this is really a bug

- [X] I confirm

### Reproduction link

https://jsfiddle.net/5vLocsjb/

### Bug description

Function addSlide addSlide function does not work properly because it adds plain text to the HTML instead of adding an HTML object. Has anyone found a solution to this problem that would allow the image to be inserted somewhere other than at the beginning or end of slider?

At first I thought it was an error in my template, but I made a clean instance of swiper and the result is exactly the same as the error.

https://jsfiddle.net/5vLocsjb/

### Expected Behavior

Insert slide between slide no. 1 and no.2. 

### Actual Behavior

Insert clean text, not parsing as HTML

### Swiper version

11.0.7

### Platform/Target and Browser Versions

Chrome, Windows

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a Swiper issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [ ] I'm willing to open a PR

**Comment by Blaaszkovsky** [comment]
Any news? :)

**Comment by cgrant992** [comment]
Had the same problem. It seems inserting a string with HTML doesn't work like it says in the documentation, you have to insert an actual DOM element:

https://jsfiddle.net/v63Lpt1q/
